contrib/babelfishpg_tsql/src/extendedproperty.c

```diff
@@ -1170,7 +1170,10 @@ get_extended_property_from_tuple(Relation relation, HeapTuple tuple,
 	values[2] = datumCopy(datum, false, -1);
```

Comment:
Can we fix this as well?

Reply:
All the other columns are declared as not null 
```
CREATE TABLE sys.babelfish_extended_properties (
  dbid smallint NOT NULL,
  schema_name name NOT NULL,
  major_name name NOT NULL,
  minor_name name NOT NULL,
  type sys.varchar(50) NOT NULL,
  name sys.sysname NOT NULL,
  orig_name sys.sysname NOT NULL,
  value sys.sql_variant,
  PRIMARY KEY (dbid, type, schema_name, major_name, minor_name, name)
);
```